src/Common/nunit/CommandLineOptions.cs

```diff
@@ -274,11 +274,12 @@ protected virtual void ConfigureOptions()
                         {
                             using (var rdr = new StreamReader(fullTestListPath))
                             {
-                                while (!rdr.EndOfStream)
+                                string line;
+                                while ( (line = rdr.ReadLine()) != null)
                                 {
-                                    var line = rdr.ReadLine().Trim();
+                                    line = line.Trim();
 
-                                    if (line[0] != '#')
+                                    if (!string.IsNullOrEmpty(line) && line[0] != '#')
                                         ((List<string>)TestList).Add(line);
```

Comment:
When fixing a bug, please don't make arbitrary changes to the code to reflect your preferred style. It makes it harder to review. In this case, I have to examine several changes in order to figure out that only one line really changes how the method works.